lib/foreman_tasks/tasks/generate_task_actions.rake

```diff
@@ -0,0 +1,41 @@
+#
+# generate_task_actions.rake is a data migration task to properly fill the missing
+# values into the foreman_tasks_tasks table.
+#
+# Run "foreman-rake foreman_tasks:generate_task_actions" to generate missing values.
+
+namespace :foreman_tasks do
+  desc 'Generate missing values for action column in foreman_tasks_tasks table.'
```

Comment:
This needs i18n I believe

Reply:
There is no i18n context defined when the rake tasks are being defined. Added translations to all the other places